api/controllers/case.js

```diff
@@ -226,13 +164,11 @@ function getUpdatedCase(user, params, newCase, oldCase) {
     "funder"
   ].map(key => cond(key, as.text));
   // date
-  ["start_date", "end_date", "post_date"].map(key => cond(key, as.date));
+  ["start_date", "end_date"].map(key => cond(key, as.date));
```

Comment:
post_date will get sent up from the client from admin users when using the ui to update/change dates/authors on articles, should we keep it here for that case?

Reply:
It was duplicated here, it is already in the `if(user.isadmin)` block above.